dnf5-plugins/copr_plugin/copr.cpp

```diff
@@ -0,0 +1,286 @@
+/// TODO: copyright headers mention libdnf?
```

Comment:
I can see most of the files say: `Copyright Contributors to the libdnf project.`.  Is it OK to spell it DNF5 project nowadays?

Reply:
I'd suggest to use the license header from some of the existing file even if it's not looking right. There is a plan to do some batch renaming/change regarding these boilerplates [here](https://github.com/rpm-software-management/dnf5/issues/410).

Reply:
ok